services/traefik/docker-compose.aws.yml

```diff
@@ -24,7 +24,7 @@ services:
       - "--providers.docker.constraints=!LabelRegex(`io.simcore.zone`, `.+`)"
       - "--entryPoints.https.forwardedHeaders.insecure"
       - "--certificatesresolvers.myresolver.acme.dnschallenge.provider=route53"
-      - "--certificatesresolvers.myresolver.acme.dnschallenge.delaybeforecheck=120"
+      # - "--certificatesresolvers.myresolver.acme.dnschallenge.delaybeforecheck=120"
```

Comment:
I would advice against commenting code/configs. 
Rather delete and use version control to keep track of changes.

Reply:
This is a good remark. I have deleted the commented lines